sharepoint/sharepoint-ps/sharepoint-online/Get-SPORestrictedAccessForSitesInsights.md

```diff
@@ -0,0 +1,244 @@
+
+| Property                     | Description                                                   |
+|:-----------------------------|:--------------------------------------------------------------|
+| SiteName                     | The name of the SharePoint site.                              |
+| SiteType                     | The type of the SharePoint site.                              |
+| SiteURL                      | The URL of the SharePoint site.                               |
+| SiteSensitivity              | The sensitivity label of the SharePoint site.                 |
+| PrimaryAdmin                 | Primary admin of the SharePoint Site                          |
+| BlockDownloadPolicy          | Boolean status of Block download policy on the site.          |
+| ConditionalAccessPolicy      | Authentication context set for the Conditional access policy  |
+| RestrictedAccessControlGroups| Entra Groups configured during policy configuration           |
+
+If this cmdlet is executed with `-RACProtectedSites` `-ReportId` `-InsightsSummary` as parameter, count of sites protected with restricted access control compared to total number of sites will be displayed.
+
+If this cmdlet is executed with `-ActionsBlockedByPolicy` as parameter, it displays the status of all active and completed reports with the following properties:
+
+| Property             | Description                                                 |
+|:---------------------|:------------------------------------------------------------|
+| Id                   | The unique Id of the report.                                |
+| CreatedDateTimeInUtc | The date and time the report creation was triggered in UTC. |
+| Status               | The status of the report.                                   |
+  
+If this cmdlet is executed with `-ActionsBlockedByPolicy` `-ReportId` `-Content TopSites` as parameter, top 100 sites with highest access denials by Restricted access control will be displayed with the following properties:
+
+| Property                     | Description                                                   |
+|:-----------------------------|:--------------------------------------------------------------|
+| SiteName                     | The name of the SharePoint site.                              |
+| SiteURL                      | The URL of the SharePoint site.                               |
+| SiteSensitivity              | The sensitivity label of the SharePoint site.                 |
+| SiteType                     | The type of the SharePoint site.                              |
+| AccessDenialsCount           | Total number of access denials on the site.                   |
+| UniqueUsersBlocked           | Count of Unique users denied access on the site.              |
```

Comment:
>Unique [](http://example.com/codeflow?start=42&length=6)

please only use lowercase unless you're referencing a brand name. eg this should be "unique"

Please review this file and the other one

Reply:
Resolved